runtime/src/bank.rs

```diff
@@ -1972,6 +1972,15 @@ impl Bank {
         }
     }
 
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn set_epoch_stakes_for_test(&mut self, epoch: Epoch, stakes: Option<EpochStakes>) {
+        if let Some(stakes) = stakes {
+            self.epoch_stakes.insert(epoch, stakes);
+        } else {
+            self.epoch_stakes.remove(&epoch);
```

Comment:
seems clearer to split this up into two separate set and remove instead of one function

Reply:
Actually we don't need to remove epoch_stake in test right now, so let's add that when we really need it.